test/test_data/libvncserver.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "libvncserver",
+        "version": "0.9.13",
+        "version_strings": ["LibVNCServer 0.9.13"],
+    }
+]
+package_test_data = [
+    {
+        "url": "http://mirror.centos.org/centos/8/AppStream/x86_64/os/Packages/",
+        "package_name": "libvncserver-0.9.11-17.el8.x86_64.rpm",
+        "product": "libvncserver",
+        "version": "0.9.11",
+    },
+    {
+        "url": "https://ftp.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/9.1/All/",
+        "package_name": "libVNCServer-0.9.13.tgz",
+        "product": "libvncserver",
+        "version": "0.9.13",
+    },
+    {
+        "url": "http://ftp.br.debian.org/debian/pool/main/libv/libvncserver/",
+        "package_name": "libvncserver1_0.9.13+dfsg-3_amd64.deb",
+        "product": "libvncserver",
+        "version": "0.9.13",
+    },
+    {
+        "url": "https://download-ib01.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/Packages/l/",
+        "package_name": "libvncserver-0.9.13-11.fc35.i686.rpm",
+        "product": "libvncserver",
+        "version": "0.9.13",
+    },
+    {
+        "url": "http://archive.ubuntu.com/ubuntu/pool/main/libv/libvncserver/",
+        "package_name": "libvncserver1_0.9.13+dfsg-1ubuntu1_amd64.deb",
+        "product": "libvncserver",
+        "version": "0.9.13",
+    },
+]
```

Comment:
Do we really need 5 different packages to test?
Maybe I'm wrong but I think we only add more packages when the version strings are different in different versions or packages (like in systemd).
IMO adding couple packages with different packages and extensions is fine (maybe it's okay now that long tests take less time).

Offtopic: Glad to have you back here :D

Reply:
> Offtopic: Glad to have you back here :D

Thanks :D

I think that the 5 different packages would show that these packages are valid across all different distros, i.e., centos, fedora, debian, etc.

Reply:
I like the different distros, but usually we try to also get some different versions at the same time (might as well prove as many things as possible per test).  I'm going to go ahead and approve and merge this as is, but I'll open an issue to suggest that maybe we could swap a few versions in the future.